### ISSUE TITLE
Stop and Run the bot automatically - Is it possible?

### ISSUE THREAD

**Issue by unzer** [Fruitfella/ROBOT-binary#25]
Is it possible stop the robot after get an expect profit or loss, but run it again automatically after a few minutes?(not manually, of course....)
So, the robot switch off or on automatically...

**Comment by Fruitfella** [comment]
@unzer 
nope, but u can do a trick to make it like that with **Run after [] seconds** block.
so the bot isnt really stop but it just doesnt trade for that "few minutes (seconds)". this way ur bot will never stop unless it touch the max loss (stop loss).
